isPrime/isPrime.cpp

```diff
@@ -0,0 +1,39 @@
+//4# Prime Number
+
+#include <iostream>
+using namespace std;
+
+void main()
+{
+	int number = 0;
+	int a = 0;
+	int status = 1;
+	cout << "Please Enter a number" << endl;
+	cin >> number;
+	//a = number / 2;
+	if (number == 2)
+	{
+		cout << "Number is prime";
+	}
+	else
+	{
+
+		for (int i = 2; i <= number / 2; i++)
+		{
+			if (number % i == 0)
+			{
+				cout << "Number is not Prime." << endl;
+				status = 0;
+				break;
+			}
+		}
```

Comment:
No need for if part as it can also be handled by the loop below.
```suggestion
	for (int i = 2; i <= number / 2; i++)
	{
		if (number % i == 0)
		{
			isPrime = false;
			break;
		}
	}
```